### DISCUSSION TITLE
Lesson 6: deploy a smart contract locally using Anvil

### DISCUSSION THREAD

**Discussion by neo-t-maredi** [Cyfrin/foundry-full-course-cu#1586] [Q&A] [answered]
Hey Guys, 

So I've run into a very irritating error: 
![Screenshot 2024-04-09 121919](https://github.com/Cyfrin/foundry-full-course-f23/assets/146746647/8079238e-e029-481c-8cd8-01f2aed60624)
![Screenshot 2024-04-09 120917](https://github.com/Cyfrin/foundry-full-course-f23/assets/146746647/ce7901a6-ce46-464b-9287-e72d9342bf25)

I'm losing my mind trying to get the file path even did the no bueno "copy path" and we are still having this problem. Any help would be greatly appreciate.

**Comment by EngrPips** [comment]
Hello @neo-t-maredi , are you using `wsl` ? and also please show your working directory. I mean like i show below 

<img width="1440" alt="Screenshot 2024-04-09 at 7 31 24 AM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/95760918/31a39cd6-6355-477d-83f7-45c148c5130b">

**Reply by neo-t-maredi** [reply]
Hey Friend! Yes I am. I think I have solved the problem though. Updated my dependencies and workflow with some newer extensions and things are going swimmingly. 

Cheers!

**Reply by EngrPips** [reply, marked as answer]
Awesome, i am glad you figured it out. keep crushing them!!!!

**Reply by noallianceb** [reply]
> Hey Friend! Yes I am. I think I have solved the problem though. Updated my dependencies and workflow with some newer extensions and things are going swimmingly.
> 
> Cheers!

Could you tell me how to update them?

**Reply by EngrPips** [reply]
> > Hey Friend! Yes I am. I think I have solved the problem though. Updated my dependencies and workflow with some newer extensions and things are going swimmingly.
> > Cheers!
> 
> Could you tell me how to update them?

What error are you getting?
